CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 * Add a method to auto-scrolling the selected region into the region list view ([#1797](https://github.com/CARTAvis/carta-frontend/issues/1797)).
+* Added cursor info for smoothed spatial profile([#1938](https://github.com/CARTAvis/carta-frontend/pull/1938)).
```

Comment:
I think this change is similar to the fourth item in Fixed: `Fixed issue where spetral profiler cursor info ...`. How about we combine them to, for example, `Show cursor info of smoothed profiles in the spatial and spectral profilers`

Reply:
Thank you for pointing that out.